src/CLICommand/common_context.ts

```diff
@@ -11,26 +11,29 @@ import {
 import { DriveID, DriveKey } from '../types';
 
 /**
- * @type {CommonContext}
+ * @type {Context}
  * A class representing the context of the parameters
  */
-export class CommonContext {
+export class Context {
```

Comment:
This name is far too general. If I just asked any of our developers randomly, "hey, what do you think a context object does?", I doubt their answer would have anything to do with parameter parsing.

Reply:
Maybe it should be called a "ParameterHelper"?

Reply:
Agreed: 65d9cb76fba080a2cd9a3783f1de88ab9afc08af